table/server/underdb/glue/src/test/java/alluxio/table/under/glue/RemoteGlueTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.table.under.glue;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.table.common.udb.UdbConfiguration;
+import alluxio.table.common.udb.UdbContext;
+
+import com.amazonaws.services.glue.AWSGlueAsync;
+import com.amazonaws.services.glue.model.GetDatabaseRequest;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RemoteGlueTest {
+
+  private static final String DB_NAME = "test";
+  private static final Map<String, String> CONF = new HashMap<>();
+
+  private UdbContext mUdbContext;
+  private UdbConfiguration mUdbConfiguration;
+  private GlueDatabase mGlueDatabase;
+  private AWSGlueAsync mGlueClient;
+
+  private static final String AWS_ACCESS_KEY_ID = "<PUT_YOUR_ACCESS_KEY_ID_HERE>";
+  private static final String AWS_SECRET_KEY = "<PUT_YOUR_SECRET_KEY_HERE>";
+  private static final String CATALOG_ID = "<PUT_YOUR_CATALOG_ID_HERE>";
+  private static final String AWS_REGION = "<PUT_GLUE_REGION_HERE>";
+
+  @Ignore
```

Comment:
instead of ignoring each method, can you ignore the entire class?

Reply:
Changed.